Olympus.FNA/Scenes/LoennScene.cs

```diff
@@ -0,0 +1,698 @@
+                        chan.Writer.Complete();
+                        return;
+                    }
+
+                    Directory.Delete(Config.Instance.LoennInstallDirectory, recursive: true);
+                    
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) {
+                        if (!string.IsNullOrEmpty(Config.Instance.LoennLinuxDesktopEntry) && File.Exists(Config.Instance.LoennLinuxDesktopEntry))
+                            File.Delete(Config.Instance.LoennLinuxDesktopEntry);
+                        if (!string.IsNullOrEmpty(Config.Instance.LoennLinuxDesktopIcon) && File.Exists(Config.Instance.LoennLinuxDesktopIcon))
+                            File.Delete(Config.Instance.LoennLinuxDesktopIcon);
+
+                        Config.Instance.LoennLinuxDesktopEntry = null;
+                        Config.Instance.LoennLinuxDesktopIcon = null;
+                    }
+                    
+                    chan.Writer.TryWrite(("Lönn successfully uninstalled!", 1f));
+
+                    Config.Instance.CurrentLoennVersion = null;
+                    Config.Instance.LoennInstallDirectory = null;
+                    Config.Instance.Save();
+                    if (updateButtons != null) await updateButtons();
+                    if (updateLabels != null) await updateLabels();
+                } catch (Exception e) {
+                    chan.Writer.TryWrite((e.ToString(), -1));
+                    chan.Writer.TryWrite(("Failed to uninstall Lönn!", -1f));
+                    AppLogger.Log.Error(e, e.Message);
+                }
+                
+                chan.Writer.Complete();
+            });
+            
+            while (await chan.Reader.WaitToReadAsync())
+            while (chan.Reader.TryRead(out (string, float) item)) {
+                if (item.Item2 >= 0) {
+                    yield return new EverestInstaller.Status(item.Item1, item.Item2,
+                        item.Item2 != 1f
+                            ? EverestInstaller.Status.Stage.InProgress
+                            : EverestInstaller.Status.Stage.Success);
+                } else {
+                    yield return new EverestInstaller.Status(item.Item1, 1f, EverestInstaller.Status.Stage.Fail);
+                }
+            }
+        }
+
+        Scener.Set<WorkingOnItScene>(new WorkingOnItScene.Job(UninstallFunc, "download_rot"), "download_rot");
+    }
+
+    public partial class PlayButton : HomeScene.IconButton {
+        public new static readonly Style DefaultStyle = new() {
+            {
+                StyleKeys.Normal,
+                new Style() {
+                    { Panel.StyleKeys.Background, new Color(0x00, 0x30, 0x10, 0x50) },
+                    { Panel.StyleKeys.Shadow, 0f },
+                }
+            },
+
+            {
+                StyleKeys.Disabled,
+                new Style() {
+                    { Panel.StyleKeys.Background, new Color(0x05, 0x35, 0x1E, 0x70) },
```

Comment:
As I said, I dont really like the current colors either..
Ill just copy from the other play button for(I originally changed from that because i thought dark green would look better, but oh well..)